apps/o2-blocks/src/todo/editor.js

```diff
@@ -1,4 +1,6 @@
 import { registerBlockType } from '@wordpress/blocks';
+// this block is deprecated and not worth the trouble to fix.
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Do we need these linting changes for `apps/o2-blocks` in this PR? From a personal perspective, I think they are adding noise to the PR, but if we do keep the changes, then we need to document that explicitly in the PR description.